migrations/versions/rev-2019-08-12-13:13:21-24e068e57321_.py

```diff
@@ -0,0 +1,30 @@
+"""empty message
+
+Revision ID: 24e068e57321
+Revises: 246b5b6123af
+Create Date: 2019-08-12 13:13:21.050238
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = '24e068e57321'
+down_revision = '246b5b6123af'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('events', sa.Column('can_pay_by_paytm', sa.Boolean(), nullable=True))
```

Comment:
nullable = False.

Reply:
Just following constraints of other columns to maintain consistency.